### ISSUE TITLE
[Screen Reader-Microbit Menu Controls] Focus order is not logical in scan mode of narrator at Fly outs of micro bit menu controls.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2593]
**User Experience:**
Users with Visual disabilities might be confuse if focus order is not logical in scan mode of narrator. Like in the scenario focus order is not logical in scan mode at micro bit controls fly outs.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1-Navigate to url:- https://makecode.com/
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate any one of the menu controls listed near the Simulator control and select it.
4-Navigate to the fly outs in scan mode(Caps + Space) of narrator and verify the issue

**Actual Result**
Focus order is not logical in scan mode of narrator at Fly outs of micro bit menu controls.

**Expected Result**
Focus order should be logical in scan mode of narrator at Fly outs of micro bit menu controls. It should be top to bottom and left to right

**Note**
Same behavior in item mode(Caps + Left/Right arrow Keys) of Narrator.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%204.2.10%20%E2%80%93%20Tracking%20of%20Focus%20and%20Selection%20Attributes.docx?d=wb24201d1227644718183e1e5353031ad&csf=1

**Refer Attachment**
[MAS 4.2.10_Makecode_Focus order is not logical in item mode.zip](https://github.com/Microsoft/pxt/files/1189686/MAS.4.2.10_Makecode_Focus.order.is.not.logical.in.item.mode.zip)

[MAS 4.2.10_MakeCode_Focus order in Scan mode of narrator.zip](https://github.com/Microsoft/pxt/files/1196678/MAS.4.2.10_MakeCode_Focus.order.in.Scan.mode.of.narrator.zip)

**Comment by veler** [comment]
This is part of Blocky, a third-part component.

**Comment by abchatra** [comment]
We have got exception for Blockly bugs from Julia Liuson (CVP). Closing these as Won't Fix. Following is the reasoning.

----------------------

MakeCode can be used in either Text editor or Block editor.

Blocks editor is inherently visual and is based on google blockly. Usage requires drag and drop functionality and we don’t have a equivalence in MakeCode for this w.r.t accessibility. 
We have evaluated couple of research projects which tries to find a equivalence to block based coding for accessibility needs: 
•	Project Torino (Requires students to buy physical blocks)
•	Accessible blockly (This is text based coding without syntax errors)
Both these are drastic departure from the blockly and provides a complete new experience. These are still in research and we intend not to use them in MakeCode at this point of time.

As workaround, the text editor in MakeCode is the way we see students with the need of accessibility using MakeCode.
